--- a/evaluator.go
+++ b/evaluator.go
@@ -184,7 +184,7 @@
 	}
 
 	teardown()
-	if err := v.Render(ctx); err != nil {
+	if err := v.Render(context.Background()); err != nil {
 		return []error{err}
 	}
 	return nil
--- a/vhs.go
+++ b/vhs.go
@@ -561,6 +561,7 @@
 	cmds = append(cmds, MakeWebM(ctx, vhs.Options.Video))
 	cmds = append(cmds, MakeScreenshots(ctx, vhs.Options.Screenshot)...)
 
+	var lastErr error
 	for _, cmd := range cmds {
 		if cmd == nil {
 			continue
@@ -568,10 +569,11 @@
 		out, err := cmd.CombinedOutput()
 		if err != nil {
 			log.Println(string(out))
+			lastErr = err
 		}
 	}
 
-	return nil
+	return lastErr
 }
 
 // ApplyLoopOffset by modifying frame sequence.
